releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,14 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 4.0.0-preview - July 23, 2020
+* Added `IConfigurationRefresher.SetDirty` method  to set the cached value for key-values registered for refresh as dirty. For users that use push notifications from event grid to receive configuration change updates, this allows the cached values to be revalidated on the next call to `RefreshAsync` or `TryRefreshAsync`.
+* Added support for deserializing configuration settings with JSON content type. [#191](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/191)
+* Added an interface `IConfigurationRefresherProvider` to provide access to instances of `IConfigurationRefresher` through dependency injection. This can be configured by adding a call to the following extension method on the service collection. [#167](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/167)
+   ```` csharp
+   public static IServiceCollection AddAzureAppConfiguration(this IServiceCollection services)
+   ````
```

Comment:
We should give a full picture before diving into specific APIs. How about this?

Added support for applications to obtain `IConfigurationRefresher` instances from the dependency injection (DI), so applications have better control of when to call `RefreshAsync()`/`TryRefreshAsync()` or whether to `await` the call to complete.

The following two APIs are introduced. To take advantage of this feature, applications need to call `IServiceCollection.AddAzureAppConfiguration()` first to add required services to the DI and then `IConfigurationRefresher` instances are accessible via `IConfigurationRefresherProvider` obtained through DI.
```cs
public static IServiceCollection AddAzureAppConfiguration(this IServiceCollection services)
```
```cs
public interface IConfigurationRefresherProvider
```